lib/smart_answer/helpers/format_capture_helper.rb

```diff
@@ -0,0 +1,64 @@
+module SmartAnswer
+  module Helpers::FormatCaptureHelper
```

Comment:
Ah yeah good point! I'll try and re-organise this better - I think I was just trying to the code in to another file, because the erb_renderer.rb was getting super long.

Reply:
I've moved them under the ErbRenderer namespace.